### PR TITLE
br: PiTR restore batch apply kv-files

### PR DESCRIPTION
**Pull request by joccau**
### What problem does this PR solve?

Issue Number: close https://github.com/pingcap/tidb/issues/38211

Problem Summary:

### What is changed and how it works?
- PITR support applying kv-files into tikv with batch method.

### Check List

Tests 

- [x] Unit test
- [ ] Integration test
- [x] Manual test (add detailed scripts or steps below)
- [ ] No code

Test report: https://pingcap.feishu.cn/docx/H5dIdJB69oLYO5xuYhscSZ7Xnic

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
Support restoring a batch of kv-files when pitr to down-cluster.
```

### CODE REVIEW

File: br/pkg/restore/import.go [hunk 1]
```diff
@@ -380,39 +384,92 @@ func (importer *FileImporter) ClearFiles(ctx context.Context, pdClient pd.Client
...
 
-	log.Debug("download and apply file done",
-		zap.String("file", file.Path),
-		zap.Stringer("take", time.Since(startTime)),
-		logutil.Key("fileStart", file.StartKey),
-		logutil.Key("fileEnd", file.EndKey),
-	)
+	for _, file := range files {
+		log.Debug("download and apply file done",
+			zap.String("file", file.Path), zap.Stringer("take", time.Since(startTime)),
```
Comment (Leavrth) [inline br/pkg/restore/import.go:470]: it seems enough to call `time.Since` once.

**Review by Leavrth — APPROVED** [review]

**Review by YuJuncen — APPROVED** [review]

File: br/pkg/restore/import.go (hunk 1 above)
Comment (YuJuncen) [reply br/pkg/restore/import.go:470]: (BTW, I think we'd better don't execute the whole for block if the log level is less than debug)

**Pull request merged**